zipkin4net/Criteo.Profiling.Tracing.UTest/Dispatchers/T_AsyncDispatcher.cs

```diff
@@ -76,9 +76,10 @@ public void RecordShouldnotBeDispatchedIfStopped()
         public void RecordsShouldBeDispatchedInOrder()
         {
             var sync = new CountdownEvent(2);
-
-            var firstRecord = new Record(new SpanState(1, 0, 1, SpanFlags.None), TimeUtils.UtcNow, Annotations.ClientRecv());
-            var secondRecord = new Record(new SpanState(1, 0, 1, SpanFlags.None), TimeUtils.UtcNow, Annotations.ClientRecv());
+            var traceId = 1;
+            var traceIdHigh = 2;
+            var firstRecord = new Record(new SpanState(traceId, traceIdHigh, 0, 1, SpanFlags.None), TimeUtils.UtcNow, Annotations.ClientRecv());
```

Comment:
nit I usually try to order traceIdHigh first, as it aligns with the hex encoding of traceIdHigh+traceId